### DISCUSSION TITLE
一分钟读论文：《软件架构对研发⽣产⼒的影响》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#10] [Announcements]
# 一分钟读论文：《软件架构对研发⽣产⼒的影响》

匈牙利塞格德⼤学科学与信息学院软件⼯程系的论文《The impact of the software architecture on the developer productivity》 基于一个 5,000 多个⼯时，⻓达 3 年的真实远程医疗应⽤研发的数据集，对四种不同的软件架构和框架组合进行了对比。

https://unbug.github.io/The-impact-of-the-software-architecture-on-the-developer-productivity/

**Comment by unbug** [comment]
3年，8倍提升。这就是好架构的重要性！
